CHANGELOG.md

```diff
@@ -269,8 +269,31 @@ Other minor additions
   toℚᵘ-homo-1/ : toℚᵘ (1/ p) ℚᵘ.≃ ℚᵘ.1/ (toℚᵘ p)
   *-inverseˡ   : 1/ p * p ≡ 1ℚ
   *-inverseʳ   : p * 1/ p ≡ 1ℚ
+  
+  positive⇒nonNegative : ∀ {q} → Positive q → NonNegative q
```

Comment:
In the CHANGELOG for brevity we usually get rid of the variables being quantified over in the type, e.g. `∀ {q} → Positive q → NonNegative q` becomes `Positive q → NonNegative q`